src/servers/main.cc

```diff
@@ -435,11 +436,15 @@ std::vector<Option> options_
        Option::ArgInt,
        "The number of threads used to accelerate copies and other operations "
        "required to manage input and output tensor contents. Default is 0."},
+      {OPTION_BACKEND_CONFIG, "backend-config", "<string>,<string>=<string>",
+       "Specify a backend-specific configuration setting. The format of this "
+       "flag is --backend-config=<backend_name>,<setting>=<value>. Where "
+       "<backend_name> is the name of the backend, such as 'tensorrt'."},
   {
-    OPTION_BACKEND_CONFIG, "backend-config", "<string>,<string>=<string>",
-        "Specify a backend-specific configuration setting. The format of this "
-        "flag is --backend-config=<backend_name>,<setting>=<value>. Where "
-        "<backend_name> is the name of the backend, such as 'tensorrt'."
+    OPTION_HOST_POLICY, "host-policy", "<string>,<string>=<string>",
+        "Specify a host policy setting associated with a policy name. The "
+        "format of this flag is --host-policy=<policy_name>,<setting>=<value>."
+        "Currently supported settings are 'numa-node', 'cpu-cores'"
```

Comment:
I think we will need to add a section to "Optimization" docs to explain how to use numa.